CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ Internal:
 - The sass-lint config has been updated to prevent the use of `@extend` with
   concrete classes.
   (PR [#550](https://github.com/alphagov/govuk-frontend/pull/550)
+- Simplify `src/` folder structure, remove tree and string manipulations from
```

Comment:
Believe this is a breaking change due to renaming 'oldie' to 'old-ie' so let's move this?

Reply:
that file was never published, so dunno. can move the log entry anyway

Reply:
ah okay, no worries then.